docs/cli.md

```diff
@@ -1,70 +1,23 @@
 # Service Catalog CLI
 
-This is a command-line interface (CLI) for interacting with
-[Kubernetes Service Catalog](../../README.md)
+This is a command-line interface (CLI) for interacting with Service Catalog
 resources. svcat is a domain-specific tool to make interacting with the Service Catalog easier.
 While many of its commands have analogs to `kubectl`, our goal is to streamline and optimize
-the operator experience, contributing useful code back upstream to Kubernetes where applicable.
+the operator experience.
 
 svcat communicates with the Service Catalog API through the [aggregated API][agg-api] endpoint on a
 Kubernetes cluster.
 
 [agg-api]: https://kubernetes.io/docs/concepts/api-extension/apiserver-aggregation/
 
-# Prerequisites
-In order to use svcat, you will need:
-
-* A Kubernetes cluster running v1.7+ or higher
-* A broker compatible with the Open Service Broker API installed on the cluster, such as:
-  * [User Provided Service Broker](../../charts/ups-broker)
-  * [Open Service Broke for Azure](https://github.com/Azure/helm-charts/tree/master/open-service-broker-azure)
-* The [Service Catalog](../../docs/install.md) installed on the cluster.
-
-# Install
-Follow the appropriate instructions for your shell to download svcat. The binary
-can be used by itself, or as kubectl plugin.
-
-## MacOS
-```
-curl -sLO https://download.svcat.sh/cli/latest/darwin/amd64/svcat
-chmod +x ./svcat
-mv ./svcat /usr/local/bin/
-svcat --version
-```
-
-## Linux
-```
-curl -sLO https://download.svcat.sh/cli/latest/linux/amd64/svcat
-chmod +x ./svcat
-mv ./svcat /usr/local/bin/
-svcat --version
-```
-
-## Windows
-
-```
-iwr 'https://download.svcat.sh/cli/latest/windows/amd64/svcat.exe' -UseBasicParsing -OutFile svcat.exe
-mkdir -f ~\bin
-$env:PATH += ";${pwd}\bin"
-svcat --version
-```
-
-The snippet above adds a directory to your PATH for the current session only.
-You will need to find a permanent location for it and add it to your PATH.
-
-## Manual
-1. Download the appropriate binary for your operating system:
-    * macOS: https://download.svcat.sh/cli/latest/darwin/amd64/svcat
-    * Windows: https://download.svcat.sh/cli/latest/windows/amd64/svcat.exe
-    * Linux: https://download.svcat.sh/cli/latest/linux/amd64/svcat
-1. Make the binary executable.
-1. Move the binary to a directory on your PATH.
+This document assumes that you've installed Service Catalog and the Service Catalog CLI
+onto your cluster. If you haven't, please see [install.md](install.md).
 
 ## Plugin
 To use svcat as a plugin, run the following command after downloading:
```

Comment:
I would maybe say "To use svcat as a kubectl plugin", just so people know what we're talking about, esp. with plugins still an alpha feature.